internal/cephfs/volumeoptions.go

```diff
@@ -275,26 +275,24 @@ func newVolumeOptionsFromVolID(ctx context.Context, volID string, volOpt, secret
 	volOptions.RequestName = imageAttributes.RequestName
 	vid.FsSubvolName = imageAttributes.ImageName
 
-	if volOpt != nil {
-		if err = extractOptionalOption(&volOptions.Pool, "pool", volOpt); err != nil {
-			return nil, nil, err
-		}
+	if err = extractOptionalOption(&volOptions.Pool, "pool", volOpt); err != nil && volOpt != nil {
```

Comment:
I assume at least one block of nested if should be allowed. for all cases, we are comparing volOpt  now

Reply:
Since it contains 5 ifs inside it, nestif calculates complexity to be 5 :( ... Should I try a different approach?

Reply:
what is the default complexity number for nested if? we can increase it as per our required for now

Reply:
Default is 4